ansible/ansible-ai-connect-service
# docker compose refined

This PR does not need a corresponding Jira item.

## Description


Eliminate duplicated compose file by using `--profile`

## Testing

### Steps to test
1. Pull down the PR
2. run `make docker-compose` and check if wisdom-service and db pods are up
3. run 'make start-backends` and check if db, prometheus and grafana pods are up

### Scenarios tested


## Production deployment
